custom/icds_reports/utils/aggregation_helpers/distributed/thr_form_v2.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_THR_V2_TABLE
+from custom.icds_reports.utils.aggregation_helpers import month_formatter
+from custom.icds_reports.utils.aggregation_helpers.monolith.base import BaseICDSAggregationHelper
+
+
+class THRFormV2AggDistributedHelper(BaseICDSAggregationHelper):
+    helper_key = 'thr-form-v2'
+    ucr_data_source_id = 'static-thr_forms_v2'
+    aggregate_parent_table = AGG_THR_V2_TABLE
+    aggregate_child_table_prefix = 'icds_db_thr_form_v2_'
+
+    def aggregate(self, cursor):
+        drop_query = self.drop_table_query()
```

Comment:
We dont use partitioned tables in citus. You might want to look to the other thr agg tables as a guide but the process should be delete then agg, not drop table, create table, then agg.

Reply:
@calellowitz  corrected.